src/site/stages/build/index.js

```diff
@@ -53,25 +88,31 @@ function defaultBuild(BUILD_OPTIONS) {
     enabledFeatureFlags: BUILD_OPTIONS.cmsFeatureFlags,
   });
 
-  smith.use(createReactPages(BUILD_OPTIONS));
-  smith.use(getDrupalContent(BUILD_OPTIONS));
-  smith.use(addDrupalPrefix(BUILD_OPTIONS));
-  smith.use(createOutreachAssetsData(BUILD_OPTIONS));
+  smith.use(createReactPages(BUILD_OPTIONS), 'Create React pages');
+  smith.use(getDrupalContent(BUILD_OPTIONS), 'Get Drupal content');
+  smith.use(addDrupalPrefix(BUILD_OPTIONS), 'Add Drupal Prefix');
+  smith.use(
+    createOutreachAssetsData(BUILD_OPTIONS),
+    'Create Outreach Assets Data',
+  );
 
-  smith.use(createEnvironmentFilter(BUILD_OPTIONS));
+  smith.use(
+    createEnvironmentFilter(BUILD_OPTIONS),
+    'Create environment filter',
+  );
 
   // This adds the filename into the "entry" that is passed to other plugins. Without this errors
   // during templating end up not showing which file they came from. Load it very early in in the
   // plugin chain.
-  smith.use(filenames());
+  smith.use(filenames(), 'Add filenames for debugging');
```

Comment:
Do we actually use `filenames`? 🤷‍♂

Reply:
Great question! I have no idea, honestly. When I was making that chart of this pipeline, it took me a bit to figure out what we _might_ be using this for, and all I could come up with was debugging. :man_shrugging: